runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.tags.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, useState, useRef, useLayoutEffect } from 'react';
+
+import If from '#src-app/components/utils/If';
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import {
+    StyledTypography,
+    TagBox,
+    DividerBox,
+    DividerAction,
+    DividerLine,
+    StaticLine,
+    Container,
+    StyledExpandIcon,
+    StyledChipName
+} from './ProcessTile.tags.styles';
+import { ProcessTileTagsProps } from './ProcessTile.types';
+
+const TAGS_CONTAINER_MARGIN_VALUE = 85;
+const TAG_RIGHT_MARGIN_VALUE = 8;
+
+const ProcessTileTags: FC<ProcessTileTagsProps> = ({ tags }) => {
+    const { translate } = useTranslations();
+
+    const [areTagsExpanded, setAreTagsExpanded] = useState<boolean>(false);
+    const [areTagsFittingLine, setAreTagsFittingLine] = useState<boolean>(false);
+    const [tagsWidth, setTagsWidth] = useState<number>(0);
+
+    const refTags = useRef(Array.from({ length: tags.length }, _ => React.createRef<HTMLDivElement>()));
+    const refTagBox = useRef<HTMLDivElement>();
+
+    const willTagsFit = (tagsWidthSum: number): void => {
```

Comment:
What do you think about changing the name to checkTagsWidth, imo it currently suggests that this function returns true or false.

Reply:
+1

Reply:
+1

Reply:
I have to admit, that it will be more similar with variables that this function has, so I think will fit. I will change it.